### PR TITLE
Add covjson-reader

### PR DESCRIPTION
**Pull request by neothemachine**

### CODE REVIEW

File: files/covjson-reader/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "covjson-reader",
+  "repo": "Reading-eScience-Centre/covjson-reader",
+  "files": {
+    "include": ["*.js"]
```
Comment (jimaek) [inline files/covjson-reader/update.json]: `*.min.js` is better for production

Comment (neothemachine) [inline files/covjson-reader/update.json]: So only minified files should be included? This doesn't match the examples in the documentation. What about source maps? They need the unminified file as well, right?

Comment (jimaek) [inline files/covjson-reader/update.json]: The ideal is to host only the minified file and nothing else. But its up to you if you want to add map files or anything else. I try to push for the min only in most cases

Comment (neothemachine) [inline files/covjson-reader/update.json]: Ok, I changed it. No problem.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**